webcompat/views.py

```diff
@@ -20,6 +20,7 @@
 from flask import session
 from flask import url_for
 from flask_firehose import push
+from flask_github import GitHubError
```

Comment:
*Thoughts*: 

I wonder if this project is still maintained. 
https://github.com/cenkalti/github-flask

Last PR on it was in 2020. and it is said to be compatible with Python 3.4
But I don't have any ideas without researching. This is not related to this PR.

Reply:
I was thinking that too. We can revisit the use of this package in the near future. There are two alternatives that I found, https://github.com/singingwolfboy/flask-dance  and authlib (https://docs.authlib.org/en/latest/client/flask.html#flask-client)